CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 <a name="Pending Release"></a>
-## [Pending Release](https://github.com/lightstep/lightstep-tracer-java/compare/master...0.13.0)
+## [Pending Release](https://github.com/lightstep/lightstep-tracer-java/compare/master...0.14.0)
+
+<a name="0.14.0"></a>
```

Comment:
It's my understanding that this is a breaking change and customers who upgrade will be required to modify their code. I'd really like to put details here as to what they should expect and what they should do.

Reply:
added